contrib/IECoreArnold/src/IECoreArnold/UniverseBlock.cpp

```diff
@@ -62,7 +62,7 @@ void loadMetadata( const std::string &pluginPaths )
 			{
 				if( dIt->path().extension() == ".mtd" )
 				{
-					if( !AiMetaDataLoadFile( dIt->path().c_str() ) )
+					if( !AiMetaDataLoadFile( (const char*)dIt->path().c_str() ) )
```

Comment:
I'm a bit confused by this one, because I expect `c_str()` to already be a `const char *`. Is it some other type on Windows, perhaps a wide char? If so, is this cast really legit?

Reply:
You know what, this might definitely be the case as Arnold isn't loading in shaders from the menu currently under Windows, this might explain things...

Reply:
Yes, this is a case of Boost on Windows returning a wide string. The most straightforward solution is to just replace .c_str() with .string().c_str(). Boost does a conversion to std::string within the string() function that plain c_str() doesn't do.

I found the Arnold SDK to build against so I'm trying this out now.